<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Log in to Facebook]]>
    </line>
    <line>
      <![CDATA[https://www.facebook.com/login/device-based/regular/login/?login_attempt=1&lwv=100]]>
    </line>
  </reporter-output>
  <suite started-at="2023-01-11T19:53:28 IST" name="Suite" finished-at="2023-01-11T19:53:39 IST" duration-ms="10761">
    <groups>
    </groups>
    <test started-at="2023-01-11T19:53:28 IST" name="Test" finished-at="2023-01-11T19:53:39 IST" duration-ms="10761">
      <class name="Assert.soft">
        <test-method is-config="true" signature="open()[pri:0, instance:Assert.soft@2f8dad04]" started-at="2023-01-11T19:53:28 IST" name="open" finished-at="2023-01-11T19:53:32 IST" duration-ms="3679" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- open -->
        <test-method signature="Run()[pri:0, instance:Assert.soft@2f8dad04]" started-at="2023-01-11T19:53:32 IST" name="Run" finished-at="2023-01-11T19:53:39 IST" duration-ms="6896" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The following asserts failed:
expected [hello] but found [Log in to Facebook]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [hello] but found [Log in to Facebook]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Assert.soft.Run(soft.java:41)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Log in to Facebook]]>
            </line>
            <line>
              <![CDATA[https://www.facebook.com/login/device-based/regular/login/?login_attempt=1&lwv=100]]>
            </line>
          </reporter-output>
        </test-method> <!-- Run -->
        <test-method is-config="true" signature="close()[pri:0, instance:Assert.soft@2f8dad04]" started-at="2023-01-11T19:53:39 IST" name="close" finished-at="2023-01-11T19:53:39 IST" duration-ms="142" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- Assert.soft -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
